src/chartmuseum/values.yaml

```diff
@@ -69,7 +69,7 @@ env:
     # disable use of index-cache.yaml
     DISABLE_STATEFILES: false
     # disable Prometheus metrics
-    DISABLE_METRICS: true
+    ENABLE_METRICS: false
```

Comment:
We should also update the [README](https://github.com/chartmuseum/charts/blame/main/src/chartmuseum/README.md#L140)

Reply:
done